test/app/index.js

```diff
@@ -56,6 +53,63 @@ let observer = new spaniel.SpanielObserver(function(changes) {
     label: 'impressed',
     ratio: 0.5,
     time: 1000
-  }]
+  }],
+  ALLOW_CACHED_SCHEDULER: true
 });
 observer.observe(target);
+
+
+// !!! CUSTOM ROOT ELEMENT !!! //
+
+
+// SpanielObserver with a custom root element
+var root = document.getElementById('root');
+
+// Watcher with a custom root element
+window.rootWatcher = new spaniel.Watcher({
+  time: 100,
+  ratio: 0.8,
+  root: root
+});
+
+var rootTarget = document.querySelector('.tracked-item-root[data-root-target-id="5"]');
+var rootObserver = new spaniel.SpanielObserver(function(changes) {
+  console.log(changes[0]);
+}, {
+  root: root,
+  rootMargin: '0px 0px',
+  threshold: [{
+    label: 'impressed',
+    ratio: 0.5,
+    time: 1000
+  }],
+  ALLOW_CACHED_SCHEDULER: true
+});
+
+root.addEventListener('scroll', () => spaniel.invalidate(), false);
```

Comment:
this doesn't look like it is removed if the observer is disposed

Reply:
I realize this is a test, but is this the pattern that is encouraged?

Reply:
Just as the consuming app will need to handle binding the event they are also responsible for removing it. In `Ember-Spaniel` for example, we will provide a clear API to do both (on/off) within Embers lifecycle hooks.